modules/post/windows/gather/psreadline_history.rb

```diff
@@ -0,0 +1,29 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/windows/user_profiles'
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info={})
+    super(update_info(info,
+        'Name' => 'Windows Gather PSReadline history',
+        'Description' => %q{
+          Gathers Power Shell history data from the target machine.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Garvit Dewan <d.garvit[at]gmail.com>' # @dgarvit
+        ],
+        'Platform' => %w{ win },
+        'SessionTypes' => [ 'meterpreter' ]
+      ))
+  end
+
+  def run
+
```

Comment:
Generally, we PR something WIP when it works in at least one specific test case, but there's nothing here. Are there commits you haven't pushed? :)

Reply:
oh, i just meant work in progress. I'll remove the tag. thanks for pointing out!

Reply:
No, it's still WIP. But you've PR'd empty code. :)

Reply:
Okay, I'll close this for now. I'll create a PR when it works in at least one specific test case. :)

Reply:
Sounds good to me! We just want something that's testable. :)